### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by Meekb** [Meekb/Overlook#24]
A user should be able to run Lighthouse and see 100% on the accessibility

![Screen Shot 2021-06-15 at 7 58 47 PM](https://user-images.githubusercontent.com/76264735/122149179-e4514380-ce18-11eb-9385-dfbba08da884.png)
